subsystem-test/framework/src/main/java/org/jboss/as/subsystem/test/AbstractSubsystemBaseTest.java

```diff
@@ -159,22 +160,17 @@ public void testSchema() throws Exception {
         SchemaValidator.validateXML(getSubsystemXml(), schemaPath, getResolvedProperties());
     }
 
+    @Test
+    public void testGetSubsystemTemplatePath() throws Exception {
+        Assert.assertTrue(getSubsystemTemplatePaths().length == 0);
+    }
+
     /**
      * To enable a test for validation of the subsystem templates, override both {@link #getSubsystemXsdPath()} and {@link #getSubsystemTemplatePaths()}
      * then add a new test just calling this method.
      */
     protected void testSchemaOfSubsystemTemplates() throws Exception {
```

Comment:
Add a @deprecate at `testSchemaOfSubsystemTemplates()` and in `getSubsystemTemplatePaths()`